tt_eager/tt_dnn/op_library/bcast/bcast_op.hpp

```diff
@@ -44,7 +44,8 @@ operation::ProgramWithCallbacks bcast_multi_core_hw(
     const Tensor &input_tensor_b,
     const Tensor &output_tensor,
     BcastOpMath bcast_op,
-    BcastOpDim bcast_dim);
+    BcastOpDim bcast_dim,
+    bool inplace);
```

Comment:
Can you please replace this with an enum class type?
This will help to avoid a need for a comment down the line.

Reply:
Sorry @ayerofieiev-tt, I'm not following. What would you like replaced with an enum? `BcastOpDim` is already `enum class BcastOpDim { H = 0, W = 1, HW = 2 };`

Reply:
I refer to the `bool inplace`. 

Reader is left guessing what true/false means in bcast_multi_core_hw(input, output, dim, true).

Being mindful of readers of your own code in this PR, you left a comment near this argument `bcast_multi_core_hw(input, output, dim, true /*inplace*/)`

I recommend to instead resolve this with an `enum class`.

Reply:
Yeah, this makes sense. There's a bit more cleanup to do with this op, I'll do that and make that change in a subsequent PR.